lib/querystring.js

```diff
@@ -158,6 +172,10 @@ function qsEscape(str) {
   return encodeStr(str, noEscape, hexTable);
 }
 
+/**
+ * @param {string | number | bigint | boolean} v
```

Comment:
```suggestion
 * @param {string | number | bigint | boolean | symbol | undefined | null} v
```

This should accept any primitives it looks like, though it accepts object at runtime it doesn't look like it is supposed to be passed an object so I left that off.